Osunick gray text

osunick opened the pull request:
Some font updates- made speed unit a gray color closer to OEM and increased font weight for speedo to better match 2023.12 update

carleeno commented on android/app/src/main/java/app/candash/cluster/DashFragment.kt:
```diff
@@ -1175,7 +1172,7 @@ class DashFragment : Fragment() {
             binding.root.setBackgroundColor(requireContext().getColor(R.color.day_background))
             textViewsPrimary.forEach { it.setTextColor(Color.BLACK) }
-            textViewsSecondary.forEach { it.setTextColor(Color.DKGRAY) }
+            textViewsSecondary.forEach { it.setTextColor(Color.parseColor("#FF888888")) }
```
fyi you could use Color.GRAY for the same 50% gray value.

osunick commented:
Reverted the speed font back to the old one because it was terrible looking, and cleaned up colors to be the constant vs hex, and made Party Mode match the Tesla UI more with some font changes.
